xrfeitoria/actor/actor_base.py

```diff
@@ -89,6 +89,11 @@ def import_from_file(
         if scale:
             Validator.validate_vector(scale, 3)
 
+        # judge file path
+        file_path = Path(file_path).resolve()
+        if not file_path.exists():
+            raise FileNotFoundError(f'File "{file_path.as_posix()}" not found')
```

Comment:
File xxx `is` not found

Reply:
fixed in [`e02c4e4` (#8)](https://github.com/openxrlab/xrfeitoria/pull/8/commits/e02c4e4fa3e5aec00688c56c24b86fe1b9dacbac)